test/filtering/test_image_grid.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+
+import itk
+import numpy as np
+import pytest
+
+from itk_cucim.filtering import image_grid
+
+
+class TestImageGrid:
+    def setup_class(self):
+        data = Path(__file__).absolute().parent.parent / "input" / "head_mr.mha"
+        # uint8 data
+        image_u8 = itk.imread(data)
+        self.image = image_u8
+
+        Caster = itk.CastImageFilter[itk.itkImagePython.itkImageUC3,
+                                     itk.itkImagePython.itkImageF3].New()
+        # float32 data
+        self.image_f32 = Caster(image_u8)
```

Comment:
Does,

```
self.image_f32 = image_u8.astype(np.float32)
```

work here?

Reply:
Yes, that is much simpler. I didn't realize the `itkImagePython` class had an `astype` method (I see there is a `dtype` attribute as well).